crowbar_framework/config/routes.rb

```diff
@@ -81,14 +81,15 @@
   # barclamps
   get "crowbar/:controller/1.0/help", action: "help", as: "help_barclamp"
   get "crowbar/:controller/1.0/proposals/nodes", action: "nodes", as: "barclamp_nodes"
-  put "crowbar/:controller/1.0/proposals", action: "proposal_create", as: "create_proposal_barclamp"
-  get "crowbar/:controller/1.0/proposals", action: "proposals", as: "proposals_barclamp"
-  post "crowbar/:controller/1.0/proposals/commit/:id", action: "proposal_commit", as: "commit_proposal_barclamp"
+  put "crowbar/:controller/1.0/proposals(.:format)", action: "proposal_create", as: "create_proposal_barclamp"
+  get "crowbar/:controller/1.0/proposals(.:format)", action: "proposals", as: "proposals_barclamp"
+  get "crowbar/:controller/1.0/proposals/template(.:format)", action: "proposal_template", as: "template_proposal_barclamp"
+  post "crowbar/:controller/1.0/proposals/commit/:id(.:format)", action: "proposal_commit", as: "commit_proposal_barclamp"
   get "crowbar/:controller/1.0/proposals/status(/:id)(/:name)(.:format)", action: "proposal_status", as: "status_proposals_barclamp"
-  delete "crowbar/:controller/1.0/proposals/:id", action: "proposal_delete", as: "delete_proposal_barclamp"
-  delete "crowbar/:controller/1.0/proposals/dequeue/:id", action: "proposal_dequeue", as: "dequeue_barclamp"
-  post "crowbar/:controller/1.0/proposals/:id", action: "proposal_update", as: "update_proposal_barclamp"
-  get "crowbar/:controller/1.0/proposals/:id", action: "proposal_show", as: "proposal_barclamp"
+  delete "crowbar/:controller/1.0/proposals/:id(.:format)", action: "proposal_delete", as: "delete_proposal_barclamp"
+  delete "crowbar/:controller/1.0/proposals/dequeue/:id(.:format)", action: "proposal_dequeue", as: "dequeue_barclamp"
+  post "crowbar/:controller/1.0/proposals/:id(.:format)", action: "proposal_update", as: "update_proposal_barclamp"
```

Comment:
Metrics/LineLength: Line is too long. [115/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)